src/harshadBJun20/assignment34/RemoveNameFromArrayList.java

```diff
@@ -0,0 +1,36 @@
+package harshadBJun20.assignment34;
+
+import java.util.ArrayList;
+
+/*Assignment - 34 :  25th July 2020
+
+Program 1 : 
+Remove all Techno from given ArrayList.
+["Maulik","Techno" ,"Krishna", "Techno", "Credits", "Techno"]
+Hint : contains & remove */
+
+public class RemoveNameFromArrayList {
+	
+	void removeName(String nameToRemove) {
+		ArrayList<String> names = new ArrayList<String>();
+		names.add("Maulik");
+		names.add("Techno");
+		names.add("Krishna");
+		names.add("Techno");
+		names.add("Credits");
+		names.add("Techno");
+		System.out.println("Given Array list:");
+		System.out.println(names);
+		System.out.println();
+		for(int index = 0;index<names.size();index++) {
```

Comment:
Your logic is correct but using while loop will reduce the iterations and if statement is also not required:
while(names.contains(nameToRemove))
				names.remove(nameToRemove);

Reply:
ok Aastha Thanks. I will do necessary changes and update the PR.

Reply:
Changes Done and updated PR as per review comment. Please review. Thanks